articles/azure/azs-how-resize-disks.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How to resize virtual machine disks using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help for resizing virtual machine disks (managed and unmanaged) using the portal on UKCloud for Microsoft Azure
+services: azure-stack
+author: Shaion O'Niel
+reviewer: BaileyLawson
+lastreviewed: 15/04/2019 11:00:00
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Resize Disks
+toc_sub3:
+toc_sub4:
+toc_title: Resize virtual machine disks - portal
+toc_fullpath: Users/How To/azs-how-resize-disks.md
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks using the UKCloud Azure Stack portal
+
+## Overview
+
+The UKCloud Azure Stack portal allows you to resize virtual machine (VM) disks. This article shows you how to resize disks that are mounted to a VM, and will work for both managed and unmanaged disks.
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack portal.
+
+## Resizing disks that are mounted to a VM
+
+1. Log in to the Azure Stack portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the favourites panel, select **Virtual machines**.
+
+    ![Virtual machines in the favourites panel](images/azsp_vmsmenu.png)
+
+3. In the *Virtual machines* blade, select the virtual machine that the disk is attached to.
+
+    ![VM Disks button](images/azs-browser-button-vm-disks.png)
```

Comment:
May be missing something subtle, but this image looks the same as the one below for step 4

Reply:
Removed it, the step itself is fairly self explanatory as well as difficult to capture the step in a screenshot